laravel-app/app/Http/Controllers/PostController.php

```diff
@@ -64,7 +81,8 @@ public function store(UserPostRequest $request): RedirectResponse
      */
     public function show(UserPost $post): View
     {
-        return view('post.show', ['post' => $post]);
+        $media = $post->media;
+        return view('post.show', ['post' => $post, 'media' => $media]);
```

Comment:
Why do you need to declare the `$media` variable if you could directly get it in the `$post`? Please revise.

Reply:
yes sir, will remove it 🙇